src/components/GoodItem/GoodItem.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+import 'bulma/css/bulma.css';
+
+interface GoodItemProps {
```

Comment:
в этом файле один компонент который принимает одни пропсы, можно легче
```suggestion
interface Props {
```